cpg_workflows/stages/large_cohort.py

```diff
@@ -25,15 +25,25 @@
 HAIL_QUERY: Final = 'hail query'
 
 
-@stage(analysis_type='combiner')
+@stage(analysis_type='combiner', analysis_keys=['vds'])
 class Combiner(CohortStage):
-    def expected_outputs(self, cohort: Cohort) -> Path:
+    def expected_outputs(self, cohort: Cohort) -> dict:
         workflow_config = config_retrieve('workflow')
         combiner_config = config_retrieve('combiner')
         output_vds_name: str = slugify(
             f"{workflow_config['cohort']}-{workflow_config['sequencing_type']}-{combiner_config['vds_version']}",
         )
-        return cohort.analysis_dataset.prefix() / 'vds' / f'{output_vds_name}.vds'
+
+        # include the list of all VDS IDs in the plan name
+        if vds_ids := config_retrieve(['combiner', 'vds_analysis_ids']):
+            ids_list_as_string = '_'.join(sorted(vds_ids))
+            combiner_plan_name = f'combiner_{ids_list_as_string}'
+        else:
+            combiner_plan_name = 'combiner'
```

Comment:
I'd add the cohort name in here as well (as soon as `cohort.name` is implemented)